runtime/amplitude/Cargo.toml

```diff
@@ -26,6 +26,7 @@ smallvec = "1.9.0"
 runtime-common = { path = "../common", default-features = false }
 
 # Custom libraries for Spacewalk
+clients-info = { git = "https://github.com/pendulum-chain/spacewalk", default-features = false}
```

Comment:
Let's also add a `rev = ... ` here. We should use `31df1832483199b978ae07f6ed4e018a11ff4dba` probably. 

To avoid confusion, let's maybe just update the `rev` value of all crates to `31df1832483199b978ae07f6ed4e018a11ff4dba` to make it consistent. There should not have been any breaking changes in the meantime.

Reply:
Yes! I didn't know what branch to use. Thanks for pointing it